membership-attribute-service/app/controllers/ExistingPaymentOptionsController.scala

```diff
@@ -20,20 +20,20 @@ import play.api.mvc.{Action, AnyContent, BaseController, ControllerComponents}
 import scalaz.-\/
 import scalaz.std.scalaFuture._
 import scalaz.syntax.monadPlus._
-import services.salesforce.ContactRepository
-import services.subscription.SubscriptionService
-import services.zuora.rest.ZuoraRestService.ObjectAccount
+import _root_.services.salesforce.ContactRepository
+import _root_.services.subscription.SubscriptionService
+import _root_.services.zuora.rest.ZuoraRestService.ObjectAccount
```

Comment:
Why have we had to do this here?

Reply:
no idea! I just did optimise imports on all the changed files as there were loads of unused imports
Let me work out what the issue is

Reply:
ok so the reason is on line 10 
`import com.gu.memsub._`
there is a `services` package in there which is then in scope.
So since it wants the services package at the root, it has to do that.

I'm inclined to just stick with what it wants, although ideally we wouldn't have two packages with the same name.

Reply:
Yea, if auto-formatting says, I normally defer to to avoid having to think about it. Good to know the reason though, thanks!